openpype/settings/defaults/project_anatomy/attributes.json

```diff
@@ -19,8 +19,7 @@
         "blender/2-91",
         "harmony/20",
         "photoshop/2021",
-        "aftereffects/2021",
-        "unreal/4-26"
```

Comment:
Seems unrelated to this PR?

Reply:
Ah, that's extra, will remove that, thanks man!